projects/1.18/assets/pyromancer/pyromancer/lang/zh_cn.json

```
@@ -1,226 +1,299 @@
-  "item.minecraft.tipped_arrow.effect.volcanic_mark_effect": "火山标记之箭",
-  "item.pyromancer.ash_projectile": "调试",
+  "item.minecraft.lingering_potion.effect.potion_of_rot": "滞留型朽腐药水",
+  "item.minecraft.potion.effect.potion_of_rot": "朽腐药水",
+  "item.minecraft.splash_potion.effect.potion_of_rot": "喷溅型朽腐药水",
+  "item.minecraft.tipped_arrow.effect.potion_of_rot": "朽腐之箭",
+  "item.pyromancer.armor_of_hellblaze_monarch_boots": "狱焰领主靴子",
+  "item.pyromancer.armor_of_hellblaze_monarch_chestplate": "狱焰领主胸甲",
+  "item.pyromancer.armor_of_hellblaze_monarch_helmet": "狱焰领主头盔",
+  "item.pyromancer.armor_of_hellblaze_monarch_leggings": "狱焰领主护腿",
+  "item.pyromancer.ash_projectile": "debug",
+  "item.pyromancer.blazing_journal.tooltip.line1": "§e将其置于炽焰台上以为其充入烈焰",
+  "item.pyromancer.blazing_journal.tooltip.line2": "§d其上魔咒会消耗烈焰",
+  "item.pyromancer.blazing_journal.tooltip.line3": "§b为此物品附魔并将其放在副手，就可使用魔咒增强攻击",
   "item.pyromancer.blazing_journal": "烈焰日志",
-  "item.pyromancer.blazing_journal.tooltip.line1": "§e放在炽焰台上以填充烈焰",
-  "item.pyromancer.blazing_journal.tooltip.line2": "§d需要烈焰的附魔",
-  "item.pyromancer.blazing_journal.tooltip.line3": "§b为此物品附魔并放在副手上以改变主手武器的攻击：",
-  "item.pyromancer.blazing_journal.tooltip.line4": "§e烈焰创伤 —— 用斧攻击着火的敌人时使敌人爆发岩浆",
-  "item.pyromancer.blazing_journal.tooltip.line5": "§e地狱丰收 —— 用锄攻击时点燃周围的敌人",
-  "item.pyromancer.blazing_journal.tooltip.line6": "§e八咫踆乌 —— 用剑攻击时将你移至敌人背后并点燃敌人",
-  "comment.yatagarasu.skdiufhiywscjjbjanc": "八咫烏／八咫ガラス，日本传说中的一种太阳鸟，这里转化为中国传说中的踆乌（太阳黑子）。其实八咫乌传说的来源据《超级大辞林》就是踆乌。",
-  "item.pyromancer.blazing_journal.tooltip.line7": "§e承接包管 —— 用锹攻击着火的敌人时使敌人昏迷",
-  "comment.undertaker.dhkakadk": "一条龙",
-  "item.pyromancer.blazing_journal.tooltip.line8": "§e灼热尖刺 —— 用钉锤攻击时将火焰穿透敌人",
-  "item.pyromancer.blazing_journal.tooltip.line9": "§e火山标记 —— 弹弓的弹射物会标记敌人，对标记过敌人的近战暴击能造成更高的伤害并点燃敌人",
-  "item.pyromancer.blazing_quill": "烈焰翎笔",
   "item.pyromancer.blazing_quill.tooltip.line1": "§d标准翎笔",
   "item.pyromancer.blazing_quill.tooltip.line2": "§e无效果",
+  "item.pyromancer.blazing_quill": "烈焰翎笔",
   "item.pyromancer.blazing_soul_spawn_egg": "烈焰浮灵刷怪蛋",
   "item.pyromancer.bombsack": "炸弹袋",
   "item.pyromancer.borov_spawn_egg": "疣狜兽刷怪蛋",
   "item.pyromancer.bottle_of_vapors": "蒸汽瓶",
+  "item.pyromancer.brimflame_tequila": "硫火龙舌兰",
   "item.pyromancer.brimstone": "硫磺石",
   "item.pyromancer.broken_evil_blade": "破碎的邪恶刀剑",
+  "item.pyromancer.burrito": "“弥留之际”玉米卷饼",
+  "item.pyromancer.call_of_flames_fire_projectile": "debug",
+  "item.pyromancer.call_of_flames.tooltip.line1": "§c以火舌焚尽敌人",
+  "item.pyromancer.call_of_flames.tooltip.line2": "§6每次使用消耗4点烈焰",
   "item.pyromancer.call_of_flames": "火召",
-  "item.pyromancer.call_of_flames.tooltip.line1": "§c焚化敌人",
-  "item.pyromancer.call_of_flames.tooltip.line2": "§6每次使用花费 4 烈焰粉",
-  "item.pyromancer.charred_unburned_spawn_egg": "炭化不燃者刷怪蛋",
   "item.pyromancer.cinnabar_chunk": "朱砂碎块",
+  "item.pyromancer.cooling_coating.tooltip.line1": "§c你的武器能给予缓慢，并对抗火的生物造成额外伤害",
```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "item.pyromancer.cooling_coating.tooltip.line1": "§c你的武器能给目标施加缓慢效果，并对抗火的生物造成额外伤害",
```
原文：**§cMakes your weapons apply Slowness and deal extra damage to fire-resistant mobs**